src/main/java/org/folio/search/service/converter/SearchDocumentConverter.java

```diff
@@ -59,8 +63,11 @@
   private SearchDocumentBody convert(ConversionContext context) {
     final var resourceData = context.getResourceData();
 
-    Map<String, Object> resultDocument = convertMapUsingResourceFields(resourceData,
+    Map<String, Object> baseFields = convertMapUsingResourceFields(resourceData,
       context.getResourceDescription().getFields(), context);
+    Map<String, Object> extendedFields = generateExtendedFields(context);
```

Comment:
I would suggest to rename it to the 'searchFields', after some time it looks most correct name for these fields, from my point of view, obviously.

Reply:
renamed to `searchFields`